index.js

```diff
@@ -1,78 +1,100 @@
-let books = [];
-const contain = document.querySelector('.list-book');
-const { localStorage } = window;
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] @AjroudiRida and @MenendezON great work implementing the changes requested, however, you still didn't code in line 1 which is disabling the linter check in your index.js file as previously requested. Also, the title in the list section should not be `Add a new book` maybe as previously suggested `All awesome books` to be in line with the content of that section.

Reply:
Hi @danmainah, thank you for your review! I just have a question about the linter rules that you want us not to disable. if you check the exercise link https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md they mentioned that we are allowed to disable some linter rules like using two classes in one file and unused var 
![Screenshot 2023-07-04 165926](https://github.com/MenendezON/W2-AwesomeBooks/assets/114816909/92dc6d6e-15dd-4040-aad4-697c47cdbd92)
So can you please explain why we can't do that in this pull request as I can see in the exercise it's clear they mentioned that we can disable the max-classes-per-file linter rule.